src/fmu/sumo/uploader/_sumofile.py

```diff
@@ -89,6 +89,7 @@ def _upload_byte_string(self, sumo_connection, object_id, blob_url):
         return response
 
     def _delete_metadata(self, sumo_connection, object_id):
+        logger.info("Deleting metadata object", object_id)
```

Comment:
Denne vil vel ikke vises ved `setLevel(logging.WARNING)`?

Reply:
Changed to logger.warn